documentation/assemblies/security/assembly-securing-access.adoc

```diff
@@ -7,15 +7,26 @@
 
 [role="_abstract"]
 Secure your Kafka cluster by managing the access a client has to Kafka brokers.
-Specify configuration options to secure Kafka brokers and clients
-
 A secure connection between Kafka brokers and clients can encompass the following:
 
 * Encryption for data exchange
 * Authentication to prove identity
 * Authorization to allow or decline actions executed by users
 
-The authentication and authorization mechanisms specified for a client must match those specified for the Kafka brokers.
+In Strimzi, securing a connection involves configuring listeners and user accounts:
+
+Listener configuration:: Use the `Kafka` resource to configure listeners for client connections to Kafka brokers.
+Listeners define how clients authenticate, such as using TLS, SCRAM-SHA-512, OAuth 2.0, or custom authentication methods.
+To enhance security, configure TLS encryption to secure communication between Kafka brokers and clients.
+You can further secure TLS-based communication by specifying the supported TLS versions and cipher suites in the Kafka broker configuration.
+For an added layer of protection, you can also specify authorization methods in the listener configuration, such as simple, OAuth 2.0, OPA, or custom authorization.
+
+User Accounts:: Set up user accounts and credentials with `KafkaUser` resources in Strimzi. 
+Users represent your clients and determine how they should authenticate and authorize with the Kafka cluster. 
+The authentication and authorization mechanisms specified in the user configuration must match the Kafka configuration. 
+Additionally, define Access Control Lists (ACLs) to control user access to specific topics and actions for more fine-grained authorization.
+You can also add configuration to limit the TLS versions and cipher suites your client uses.
```

Comment:
No, you cannot do this in the user account - or at least not in KafkaUser. You can do this only in the producer / consumer configuration.

Reply:
I added a space and updated as follows to make this clear:

> You can also add producer or consumer configuration to your clients if you wish to limit the TLS versions and cipher suites they use. 
> The configuration on the client must only use protocols and cipher suites that are enabled on the broker.